README.md

```diff
@@ -34,9 +34,10 @@ The application can either be run on your local machine or built inside a docker
 You'll need to set the following env vars to run the application:
 
 ```
-NOTIFY_KEY       | Your GOV.UK notify key
-ICASEWORK_KEY    | Key for the iCasework API
-ICASEWORK_SECRET | Secret for the API key
+NOTIFY_KEY            | Your GOV.UK notify key
+AWS_SQS               | AWS SQS URL
+AWS_SECRET_ACCESS_KEY | AWS Secret Access Key
+AWS_ACCESS_KEY_ID     | AWS Access Key ID
 ```
```

Comment:
can you add a `WRITE_TO_CASEWORK` env variable.  I think this was missed in a previous PR

can also briefly explain what happens e.g. when this value is set to true, the form will be sent to the SQS queue